linters/pyright/pylint.test.ts

```diff
@@ -0,0 +1,13 @@
+import path from "path";
+import { customLinterCheckTest, linterCheckTest, TestCallback } from "tests";
+import { TEST_DATA } from "tests/utils";
+
+linterCheckTest({ linterName: "pyright" });
```

Comment:
`linter<type>Test` and `customLinter<type>Test` don't always interact super cleanly with snapshot generation. I think we've addressed all the problems, and this should be fine, but if you run into any problems, happy to help. You may need to rerun the test and commit a new snapshot file.